torch/fx/passes/utils/matcher_utils.py

```diff
@@ -2,14 +2,17 @@
 from collections import defaultdict
 import copy
 import torch
-from torch.fx.graph import Graph
-from torch.fx.node import Node
+from torch.fx import (
+    Node,
+    Graph,
+    GraphModule,
+)
 from torch.fx._compatibility import compatibility
 from typing import Dict, List, Set, Any, Union, Tuple
 import logging
 import os
 
-__all__ = ['SubgraphMatcher', 'InternalMatch']
+__all__ = ['SubgraphMatcher', 'InternalMatch', 'SubgraphMatcherWithNameNodesMap']
```

Comment:
I am not sure how widely useful this will be for OSS user. 

I suggest keeping this "private", or specific to quantization workflow.

Reply:
this is going to be official API for quant workflow though?

Reply:
yes, I would prefer this in a different file.